mock/ext-api-dyson/get/gochain-api-commands1.js

```diff
@@ -0,0 +1,85 @@
+/// Gochain API Mock
+/// See:
+/// curl "http://localhost:3000/gochain-api/transactions?address=0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "http://localhost:3000/gochain-api/tokens?address=0x0Fd98FB42C439E5F6484f7E71Caa6661d81d0628"
+/// curl "https://{go rpc}/transactions?address=0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "https://{go rpc}/ 
+/// curl "http://localhost:8420/v1/gochain/0xEd7F2e81B0264177e0df8f275f97Fd74Fa51A896"
+/// curl "http://localhost:8420/v2/gochain/tokens/0x0Fd98FB42C439E5F6484f7E71Caa6661d81d0628?Authorization=Bearer"
+
+module.exports = {
+    path: '/gochain-api/:command1?',
```

Comment:
What is the purpose to test callisto-api-commands1.js  and gochain-api-commands1.js  is it's the same provider? it just creates tons of duplication

Reply:
True, but end-to-end usecase is not the same even though internally it runs in the same implementation.